packages/core/src/routes/session/session.test.ts

```diff
@@ -169,6 +176,48 @@ describe('sessionRoutes', () => {
       );
     });
 
+    it('register user with admin role for admin console if no active user found', async () => {
+      interactionDetails.mockResolvedValueOnce({
+        params: { client_id: adminConsoleApplicationId },
+      });
+
+      hasActiveUsers.mockResolvedValueOnce(false);
+
+      await sessionRequest
+        .post('/session/register/username-password')
+        .send({ username: 'username', password: 'password' });
+
+      expect(insertUser).toHaveBeenCalledWith(
+        expect.objectContaining({
+          id: 'user1',
+          username: 'username',
+          passwordEncrypted: 'password_user1',
+          passwordEncryptionMethod: 'Argon2i',
+          roleNames: ['admin'],
+        })
```

Comment:
nit: in this test case, seems that there is no need to check the fields (that have been checked by the case above) except `roleNames` here when we use `expect.objectContaining({ …… })`

```suggestion
        expect.objectContaining({
          roleNames: ['admin'],
        })
```

Reply:
updated